pkg/dicomio/reader.go

```diff
@@ -67,6 +67,7 @@ type Reader interface {
 	// SetCodingSystem sets the charset.CodingSystem to be used when ReadString
 	// is called.
 	SetCodingSystem(cs charset.CodingSystem)
+	GetByteOrder() binary.ByteOrder
```

Comment:
typically in Go, we don't include `Get` in a getter's name (https://golang.org/doc/effective_go.html#Getters).

So, this can just be `ByteOrder()`